openslides/agenda/serializers.py

```diff
@@ -56,11 +52,7 @@ class Meta:
         fields = (
             'id',
             'item_number',
-            'item_no',
             'title',
```

Comment:
I am not sure whether we have to add the line `read_only_fields = ('title',)` because this is a read only property now. Have a look at our REST page.